### ISSUE TITLE
Cannot install zf2 framework because it requires this module witch needs ext-ldap

### ISSUE THREAD

**Issue by svycka** [zendframework/zend-ldap#1]
composer error:
```
$ composer require zendframework/zendframework
Using version ^2.5 for zendframework/zendframework
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - zendframework/zend-ldap 2.5.1 requires ext-ldap * -> the requested PHP extension ldap is missing from your system.
    - zendframework/zend-ldap 2.5.0 requires ext-ldap * -> the requested PHP extension ldap is missing from your system.
    - zendframework/zendframework 2.5.0 requires zendframework/zend-ldap ~2.5.0 -> satisfiable by zendframework/zend-ldap[2.5.0, 2.5.1].
    - Installation request for zendframework/zendframework ^2.5 -> satisfiable by zendframework/zendframework[2.5.0].


Installation failed, deleting ./composer.json.
```
I don't use ldap... Of course I can add all dependencies one by one but still with 2.4 there was no problem

**Comment by samsonasik** [comment]
it can be closed in favor of https://github.com/zendframework/zf2/pull/7571

**Comment by weierophinney** [comment]
Fixed with zendframework/zf2#7571 — if you update now, you will no longer be blocked by the ext-ldap requirement.
